remote-api/src/main/java/org/alfresco/rest/framework/core/ResourceInspector.java

```diff
@@ -378,20 +383,16 @@ public static int validSuccessCode(HttpMethod httpMethod, int success)
             return success;
         }
 
-        switch (httpMethod)
-        {
-            case GET:
-                return Status.STATUS_OK;
-            case POST:
-                return Status.STATUS_CREATED;
-            case PUT:
-                return Status.STATUS_OK;
-            case DELETE:
-                return Status.STATUS_NO_CONTENT;
-            default:
-                return Status.STATUS_OK;
-        }
+        if (GET.equals(httpMethod))
+            return Status.STATUS_OK;
+        if (POST.equals(httpMethod))
+            return Status.STATUS_CREATED;
+        if (PUT.equals(httpMethod))
+            return Status.STATUS_OK;
+        if (DELETE.equals(httpMethod))
+            return Status.STATUS_NO_CONTENT;
```

Comment:
We should use braces here to be compliant with rest of the class.

Reply:
Done